pkg/resmgr/nri.go

```diff
@@ -215,55 +230,64 @@ func (p *nriPlugin) CreateContainer(pod *api.PodSandbox, container *api.Containe
 	m.policy.ExportResourceData(c)
 	m.updateTopologyZones()
 
-	adjust := p.getPendingCreate(container)
-	updates := p.getPendingUpdates(container)
+	adjust = p.getPendingCreate(container)
+	update = p.getPendingUpdates(container)
 
-	p.dump("<= CreateContainer", "adjustments", adjust, "updates", updates)
-
-	return adjust, updates, nil
+	return adjust, update, nil
 }
 
 func (p *nriPlugin) StartContainer(pod *api.PodSandbox, container *api.Container) error {
-	method := "NRI-StartContainer"
-
-	p.dump(method, "pod", pod, "container", container)
+	event := "StartContainer"
+	p.dump(request, event, pod, container)
 
 	m := p.resmgr
-
 	m.Lock()
 	defer m.Unlock()
 
 	c, ok := m.cache.LookupContainer(container.Id)
-	if ok {
-		c.UpdateState(cache.ContainerStateRunning)
+	if !ok {
+		return nil
+	}
 
-		e := &events.Policy{
-			Type:   events.ContainerStarted,
-			Source: "resource-manager",
-			Data:   c,
-		}
+	c.UpdateState(cache.ContainerStateRunning)
 
-		if _, err := m.policy.HandleEvent(e); err != nil {
-			m.Error("%s: policy failed to handle event %s: %v", method, e.Type, err)
-		}
+	e := &events.Policy{
+		Type:   events.ContainerStarted,
+		Source: "resource-manager",
+		Data:   c,
 	}
 
-	// test timeouts
-	//time.Sleep(3 * time.Second)
+	if _, err := m.policy.HandleEvent(e); err != nil {
+		m.Error("%s: policy failed to handle event %s: %v", event, e.Type, err)
+	}
 
 	return nil
 }
 
-func (p *nriPlugin) UpdateContainer(pod *api.PodSandbox, container *api.Container) ([]*api.ContainerUpdate, error) {
-	p.dump("NRI-UpdateContainer", "pod", pod, "container", container)
+func (p *nriPlugin) UpdateContainer(pod *api.PodSandbox, container *api.Container) (updates []*api.ContainerUpdate, retErr error) {
+	event := "UpdateContainer"
+	p.dump(request, event, pod, container)
```

Comment:
```suggestion
func (p *nriPlugin) UpdateContainer(pod *api.PodSandbox, container *api.Container) (update []*api.ContainerUpdate, retErr error) {
	event := "UpdateContainer"
	p.dump(request, event, pod, container)
```
nit: perhaps we can use the same name for this argument across all the container management methods? Same questions for `StopContainer()`

Reply:
I'll update those to be named consistently.